src/openforms/submissions/api/viewsets.py

```diff
@@ -198,6 +200,15 @@ def _complete(self, request, *args, **kwargs):
 
         logevent.form_submit_success(submission)
 
+        rules = FormLogic.objects.filter(form=submission.form)
+        logs = TimelineLogProxy.objects.filter(
+            template="logging/events/submission_logic_evaluated.txt", 
+            object_id = submission.id)
+        for rule in rules :
+            # rule has never been triggered
+            if rule.id not in [log.extra_data['rule_id'] for log in logs]:
+                logevent.submission_logic_evaluated(submission, rule)
+
```

Comment:
@joeribekker submission completion does not trigger another logic rule evaluation (and I don't see why it should), did you request for these rules to be logged (again)? I'm not convinced this should be here.